x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/EnterpriseSearchFeatures.java

```diff
@@ -14,8 +14,17 @@
 import org.elasticsearch.xpack.application.connector.ConnectorTemplateRegistry;
 
 import java.util.Map;
+import java.util.Set;
 
 public class EnterpriseSearchFeatures implements FeatureSpecification {
+
+    public static final NodeFeature QUERY_RULES_TEST_API = new NodeFeature("query_rules.test");
```

Comment:
Nit: The pattern for adding cluster features I've seen to date defines the `NodeFeature` in the class the implements the new functionality